src/features/unified-share-modal/ContactsField.js

```diff
@@ -87,7 +92,7 @@ class ContactsField extends React.Component<Props, State> {
                 fullContacts = this.addSuggestedContacts(fullContacts);
             }
 
-            return fullContacts.map<Object>(({ email, id, isExternalUser, name, type }) => ({
+            return fullContacts.map<any>(({ email, id, isExternalUser, name, type }) => ({
```

Comment:
any reason the type here is being changed from `Object` to `any` ?

Reply:
this was pulled from the previous big PR, so i kept it. since there are a few questions around this, lemme add a more useful type to this file (This has to do with the TODO near this line)